doc/nrf/shortcuts.txt

```diff
@@ -6,7 +6,7 @@
 .. |release_tt| replace:: ``v2.9.0-nRF54H20-rc1``
 .. |release_number_tt| replace:: ``2.9.0-nRF54H20-rc1``
 
-.. |jlink_ver| replace:: v7.94i
+.. |jlink_ver| replace:: v8.10f
```

Comment:
I assume this line must be updated in addition: https://github.com/nrfconnect/sdk-nrf/blob/d899e22404f40e028cc7c1cbf71930e06e321cd3/scripts/docker/Dockerfile#L19

Correct @thst-nordic ?

Reply:
It is handled by Buran team, should be @jangalda-nsc

Reply:
as long as this just gets in sync before next release.
Deviating values is a recipe to confuse users.